app/src/main/java/com/framgia/forder/data/source/remote/ProductRemoteDataSource.java

```diff
@@ -62,4 +65,26 @@ public ProductRemoteDataSource(FOrderApi api) {
                     }
                 });
     }
+
+    @Override
+    public Observable<List<Comment>> getListCommentInProduct(int productId, int domainId) {
+        List<Comment> comments = new ArrayList<>();
+        comments.add(new Comment(1, 1, "Trần Đức Quốc", "Rất ngon", "26/04/2017"));
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "26/04/2017" appears 9 times in this file; the first occurrence is on line 72
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals